icons/filezilla/filezilla-line-wordmark.svg

```diff
@@ -0,0 +1 @@
.218-.661-.447-.98-.688-.114-.087-.418-.196-.676-.236a3.268 3.268 0 0 1-.786-.244c-.463-.24-2.395-.244-2.866-.004m53.15-.109a8.934 8.934 0 0 0-.437.182c-.174.08-.584.167-.915.197-.459.036-.622.083-.728.207-.073.087-.297.19-.494.226-.638.12-.968.633-.604.942.107.11.179.248.21.397.071.4.485.448.815.095.088-.095.33-.186.604-.23.254-.04.532-.138.619-.214.133-.124.342-.146 1.496-.168l1.34-.025.497.244c.273.134.535.244.58.244.126.003.836.699.836.819 0 .062.087.203.194.32.224.24.303.662.315 1.609.011.913.076.873-1.636.939-1.242.047-1.489.08-1.705.215-.17.105-.49.178-.98.222-.573.054-.763.105-.907.236-.099.095-.239.171-.311.171a1.342 1.342 0 0 0-.718.335.502.502 0 0 1-.284.142c-.186 0-1.564 1.329-1.637 1.58-.06.157-.14.31-.235.448-.357.524-.342 3.494.019 3.763.099.073.216.266.258.426.046.164.144.313.224.339.084.025.186.135.236.244.049.109.144.196.205.196.064 0 .224.091.356.208.137.113.38.24.543.284.179.05.342.138.486.25.281.26 3.056.267 3.406.008.125-.09.402-.207.618-.266.22-.054.445-.163.502-.25.06-.084.159-.154.22-.154.11 0 .212-.083.968-.8.588-.565.69-.492.797.56.076.724.133.765 1.185.812.6.025.74.01.797-.091.14-.251.004-10.349-.144-10.618-.073-.135-.175-.473-.228-.757-.053-.28-.163-.586-.24-.673a1.774 1.774 0 0 1-.242-.466c-.061-.168-.171-.339-.251-.379-.076-.044-.14-.116-.14-.164 0-.142-.627-.684-.9-.779a.89.89 0 0 1-.372-.24c-.084-.106-.293-.178-.676-.24a6.049 6.049 0 0 1-.976-.24c-.406-.15-2.198-.222-2.562-.106m-71.715.415-.197.058-.023 7.044-.019 7.043.22.058c.255.066 2.028.182 2.088.138.023-.018.038-3.257.038-7.2v-7.17l-.953-.015c-.523-.003-1.044.015-1.154.044m46.848.025c-.102.102-.148 13.829-.05 14.09.039.1.18.128.658.128.334.004.763.037.949.07.284.058.353.047.414-.073.083-.153.121-14.098.038-14.233-.073-.116-1.891-.102-2.009.018m-25.69 1.304a7.2 7.2 0 0 0 .474.243c.315.15 1.227 1.074 1.401 1.427.088.171.179.346.274.517.09.179.144.375.155.575 0 .168.072.506.16.754.353 1.008.539.964-4.062.979-4.507.014-4.184.08-4.002-.8.05-.22.087-.441.11-.663 0-.08.091-.255.201-.383.11-.13.198-.323.198-.43 0-.188.209-.469.698-.945.14-.139.259-.292.259-.343 0-.047.072-.112.159-.138a.652.652 0 0 0 .277-.233c.068-.098.273-.226.456-.28.186-.055.402-.168.482-.251.186-.2 2.426-.226 2.76-.033m54.919 5.66.216.16.023 1.245c.019 1.085.004 1.289-.13 1.584a3.262 3.262 0 0 0-.208.677c-.038.233-.118.382-.247.465-.106.07-.193.19-.193.266 0 .19-1.021 1.194-1.211 1.194-.05 0-.232.098-.403.219-.307.21-.338.214-1.53.218l-1.215.004-.231-.222c-.125-.12-.27-.219-.315-.219-.118 0-.657-.568-.657-.691 0-.051-.091-.193-.201-.31-.247-.266-.312-2.147-.076-2.34.064-.055.186-.252.27-.437.083-.19.223-.382.31-.43a.48.48 0 0 0 .21-.236c.026-.084.163-.197.303-.255.14-.055.3-.164.353-.237.053-.073.255-.156.448-.185.194-.033.437-.128.536-.211.224-.193.239-.2 1.662-.343 1.481-.145 2.001-.127 2.286.084" style="fill:none;stroke-width:.951649;stroke-linecap:butt;stroke-linejoin:miter;stroke:#000;stroke-opacity:1;stroke-miterlimit:4" transform="matrix(1.02893 0 0 1.07315 -1.164 -3.79)"/></g></g></svg>
```

Comment:
Looks good, but the code uses a stroke instead of a path. So you need to convert the stroke to a path.
Pretty much what's left is following [this guide](https://github.com/devicons/devicon/issues/1306#issuecomment-1297625294) from step 5. onwards